### PR TITLE
Add flag to preserve internal borders when border=False

### PR DESCRIPTION
**Pull request by myheroyuki**
Addresses issue #178

### CODE REVIEW

File: tests/test_prettytable.py [hunk 1]
```diff
@@ -1908,3 +1908,74 @@ def test_max_table_width(self):
...
+            <td style="padding-left: 1em; padding-right: 1em; text-align: center; vertical-align: top">value6</td>
+        </tr>
+        <tr>
+            <td style="padding-left: 1em; padding-right: 1em; text-align: center; vertical-align: top">value 7</td>
+            <td style="padding-left: 1em; padding-right: 1em; text-align: center; vertical-align: top">value8</td>
+            <td style="padding-left: 1em; padding-right: 1em; text-align: center; vertical-align: top">value9</td>
+        </tr>
+    </tbody>
+</table>
+""".strip()
```
Comment (hugovk) [inline tests/test_prettytable.py]:
```suggestion
""".strip()  # noqa: E501
```

File: tests/test_prettytable.py [hunk 2]
```diff
@@ -1908,3 +1908,74 @@ def test_max_table_width(self):
...
+
+        assert (
+            pt.get_string().strip()
+            == """
+ Field 1 | Field 2 | Field 3  
+---------+---------+---------
+ value 1 |  value2 |  value3  
+ value 4 |  value5 |  value6  
+ value 7 |  value8 |  value9 
+""".strip()
```
Comment (hugovk) [inline tests/test_prettytable.py]:
```suggestion
 value 7 |  value8 |  value9  
""".strip(). # noqa: W291
```

File: src/prettytable/prettytable.py [hunk 3]
```diff
@@ -211,6 +214,10 @@ def __init__(self, field_names=None, **kwargs):
             self._border = kwargs["border"]
         else:
             self._border = True
+        if kwargs["preserve_internal_border"] in (True, False):
+            self._preserve_internal_border = kwargs["preserve_internal_border"]
```
Comment (hugovk) [inline src/prettytable/prettytable.py:218]: Is this line needed? If so, please can you add tests to cover it?

File: README.md [hunk 4]
```diff
@@ -402,7 +402,9 @@ whatever you prefer. The `set_style` method just does this automatically for you
 The options are these:
 
 - `border` - A boolean option (must be `True` or `False`). Controls whether a border is
-  drawn around the table.
+  drawn inside and around the table.
+- `preserve_internal_border` - A boolean option (must be `True` or `False`). Controls
+  whether borders are still drawn within the table even when border = False.
```
Comment (hugovk) [inline README.md]:
```suggestion
  whether borders are still drawn within the table even when `border=False`.
```

**Review by hugovk — CHANGES_REQUESTED** [review]
Thanks for the PR!

**Comment by myheroyuki** [comment]
Hi @hugovk, I made the changes you requested. If there are any other changes you'd like me make just let me know. Thanks!

**Review by hugovk — APPROVED** [review]

**Comment by hugovk** [comment]
Thanks!

**Pull request merged**